main.py

```diff
@@ -12,6 +12,9 @@
 import js_main
 import py_main
 import gast_to_code.gast_to_code_main as gtc
+import firebase_admin
+from firebase_admin import credentials
```

Comment:
It should be possible to configure firebase to accept writes from lower tier auth if this file didn't use the admin package.

You'd have to configure the firestore rules to allow restricted writes, but I think it would make it safe to have on GitHub.

By default the rules will disallow all reads/writes (I'm pretty sure). Here's how to change that: https://firebase.google.com/docs/firestore/security/rules-conditions. Although not necessarily part of deployment having that rules file in the repo would be nice too.